components/eamxx/scripts/test_all_scream.py

```diff
@@ -166,6 +166,11 @@ def __init__(self, _):
             on_by_default=False,
             default_test_len="short"
         )
+        if tas is not None:
```

Comment:
Rather than storing this in the repo, why don't we store it in the $baseline_dir folder for this machine? this way we can manually go and edit the file if need be, without the need for a PR?

We could store the supp file location inside our `machine_specs.py`, so that if someone runs on their personal machine (i do on my workstation/laptop quite often), they can have their own copy (stored in `~/.cime/scream_mach_specs.py`).

Reply:
@bartgol interesting idea. There's some advantages to having the supp file be version controlled but it is probably better to have the flexibility of keeping it outside the repo.